linode/instanceip/framework_schema.go

```diff
@@ -41,6 +41,7 @@ var frameworkResourceSchema = schema.Schema{
 
 		"address": schema.StringAttribute{
 			Description: "The resulting IPv4 address.",
+			Optional:    true,
```

Comment:
Not sure why the address has become Optional.

Reply:
Also assuming 'reserved' will be added in a later PR? (If this is the response you get when listing instance IPs

Reply:
Yes the 'reserved' field is added in the other PR